.github/workflows/ci.yml

```diff
@@ -7,6 +7,36 @@ on:
   pull_request:
 
 jobs:
+  size-check:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    container:
+      image: ubuntu:latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Setup Container
+        run: apt update && apt install -y git
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - name: Install Dependencies
+        run: npm ci
+      - name: Running Prod Build
+        run: npm run cdn:build:dev
+      - name: Generating Asset Size Report
+        run: node ./tools/scripts/diff-sizes.mjs -o build
+      - name: Creating PR Comment
+        run: |
+          node ./tools/scripts/comment-pr.mjs \
+            --pull-request=${{ github.event.pull_request.number }} \
+            --token=${{ secrets.GITHUB_TOKEN }} \
```

Comment:
I can never quite remember the safety issues around these CI tasks, so I have to ask to make sure ;P, 

Is there any danger to using/exposing this in a `pull_request` type action?

Reply:
If the CI workflow runs on a PR from a fork, the secrets will be pulled from that fork and not our repo. Our secrets cannot be exposed via a fork unless we use `pull_request_target`